isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -2133,6 +2134,96 @@ namespace Isis {
   }
 
 
+/**
+ * Returns the latitude and longitude range for the Cube. More accurate than the minimum and 
+ * maximum latitude and longitude from the mapping group. 
+ * 
+ * @param minLatitude minimum latitude of the cube
```

Comment:
Why is this worded differently than the other parameters?

Reply:
Not sure why, really. Fixing this.